app/controllers/api_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ApiController < ApplicationController
+  before_action :doorkeeper_authorize!
+
+  skip_before_action :verify_authenticity_token
+
+  respond_to :json
+
+  def current_user
+    @current_user ||= User.find(doorkeeper_token.resource_owner_id)
+  end
```

Comment:
Are you sure a controller is best for this??

Reply:
don't forget what controllers are really meant for: https://www.tutorialspoint.com/ruby-on-rails/rails-controllers.htm